go.mod

```diff
@@ -40,3 +40,5 @@ replace github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v
 replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v10.15.5+incompatible
 
 replace github.com/banzaicloud/bank-vaults => github.com/banzaicloud/bank-vaults v0.0.0-20190508130850-5673d28c46bd
+
+replace github.com/jenkins-x/lighthouse => github.com/abayer/lighthouse v0.0.0-20200514182250-29f759f9c943
```

Comment:
We've got a bit of a chicken and the egg problem here - this is needed to pick up changes in Lighthouse's SCM provider client for tests to pass here, but I can't merge that PR until this is done, so...yeah.